de.tudarmstadt.ukp.dkpro.argumentation.nio/src/main/java/de/tudarmstadt/ukp/dkpro/argumentation/nio/writer/SpanAnnotationException.java

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright (c) the Department of Informatics, Technische Universität Darmstadt. All Rights Reserved.
+ *
+ * Unauthorized distribution of this file via any medium is strictly prohibited.
+ */
+package de.tudarmstadt.ukp.dkpro.argumentation.nio.writer;
+
+/**
+ * @author <a href="mailto:shore@ukp.informatik.tu-darmstadt.de">Todd Shore</a>
+ * @since May 2, 2016
+ *
+ */
+public class SpanAnnotationException extends Exception {
+
+	/**
+	 *
+	 */
+	private static final long serialVersionUID = -8088424492769878641L;
+
+	private final int begin;
+
+	private final int end;
+
+	private final String label;
+
+	/**
+	 *
+	 */
+	public SpanAnnotationException(final int begin, final int end, final String label) {
```

Comment:
The exception doesn't always refer to an annotation which actually exists: see [my other comment on the class](https://github.com/dkpro/dkpro-argumentation/pull/5#discussion_r68022158).